.code-samples.meilisearch.yaml

```diff
@@ -327,25 +324,25 @@ getting_started_search_md: |-
 
   [About this SDK](https://github.com/meilisearch/meilisearch-js/)
 get_attributes_for_faceting_1: |-
-  client.index('movies').getAttributesForFaceting()
+  client.index('movies').getFilterableAttributes()
 update_attributes_for_faceting_1: |-
```

Comment:
should be changed with `update_filterable_attributes_1` or depending on the name of the section